services/graph/pkg/validate/libregraph.go

```diff
@@ -9,6 +10,10 @@ import (
 	"github.com/owncloud/ocis/v2/services/graph/pkg/unifiedrole"
 )
 
+var (
+	_contextRoleIDsValueKey = "roleFilterIDs"
```

Comment:
context keys should not be strings to avoid clashes. This should be a custom type.